frontend/src/vuex/modules/validators.js

```diff
@@ -37,40 +47,39 @@ export default () => {
       commit("setTotalActive", data.total_active)
       commit("setTotalFound", data.totalFound)
 
-
       return data.validators
-    },
+    }
 
-    async getValidators({ commit, rootState }) {
-      commit("setLoading", true)
+    // async getValidators({ commit, rootState }) {
+    //   commit("setLoading", true)
 
-      let validators = await fetchValidators(
-        rootState.connection.networkConfig.id
-      )
-      // @ts-ignore
-      // window.validators = validators
+    //   let validators = await fetchValidators(
+    //     rootState.connection.networkConfig.id
+    //   )
+    //   // @ts-ignore
+    //   // window.validators = validators
 
-      // const { delegates } = rootState.delegates
+    //   // const { delegates } = rootState.delegates
 
-      // if (validators) {
```

Comment:
If you didn't change anything in this file - please revert it to base

Reply:
Can I revert the single file? :(